openapi.yml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: In-band Network Telemetry (INT) for EVCs
+  description: NApp to deploy In-band Network Telemetry (INT) for EVCs
+servers:
+  - url: /api/kytos/telemetry_int
+paths:
```

Comment:
The rest of the other routes will be incrementally added as new endpoints are implemented and covered.